net/apinger/Makefile

```diff
@@ -56,6 +56,14 @@ define Package/apinger/install
 	$(INSTALL_DATA) $(PKG_BUILD_DIR)/src/apinger.conf $(1)/etc/apinger.conf
 	$(INSTALL_DIR) $(1)/etc/init.d
 	$(INSTALL_BIN) ./files/apinger.init $(1)/etc/init.d/apinger
+	$(INSTALL_DIR) $(1)/etc/config
+	$(INSTALL_BIN) ./files/apinger.config $(1)/etc/config/apinger
```

Comment:
That is not a INSTALL_BIN please use INSTALL_DATA

Reply:
Updated